Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -86,9 +86,9 @@ public final class SwiftPackageManagerGraphGenerator: SwiftPackageManagerGraphGe
             let name = dependency.packageRef.name
             let packageFolder: AbsolutePath
             switch dependency.packageRef.kind {
-            case "remote":
+            case "remote", "remoteSourceControl":
                 packageFolder = checkoutsFolder.appending(component: dependency.subpath)
-            case "local":
+            case "local", "fileSystem":
                 // Depending on the swift version, the information is available either in `path` or in `location`
                 guard let path = dependency.packageRef.path ?? dependency.packageRef.location else {
                     throw SwiftPackageManagerGraphGeneratorError.missingPathInLocalSwiftPackage(name)
```

Comment:
What about `localSourceControl`? when is it used?

Reply:
I've added it back just in case, but in general it shouldn't appear.  

Given the current format of our generated `Package.swift` file (used overloads of `.package()` methods in particular), in order to get `"kind" : "localSourceControl"` here, a user would have to put this kind of strange definition in `Dependencies.swift` file:
```
.remote(url: "/Developer/APackage", requirement: .revision("bedef0"))
```

Whereas the sensible thing to do would be:
```
.local(path: "/Developer/APackage")
```
which produces `"kind" : "fileSystem"`